include/k4ainternal/k4aplugin.h

```diff
@@ -87,11 +87,21 @@ typedef enum
  */
 typedef enum
 {
-    K4A_TRANSFORM_ENGINE_TYPE_COLOR_TO_DEPTH = 0, /**< Transforms the color image into the geometry of the depth camera
-                                                   */
-    K4A_TRANSFORM_ENGINE_TYPE_DEPTH_TO_COLOR /**< Transforms the depth image into the geometry of the color camera */
+    K4A_TRANSFORM_ENGINE_TYPE_COLOR_TO_DEPTH = 0,   /**< Transforms color image into the geometry of the depth camera */
+    K4A_TRANSFORM_ENGINE_TYPE_DEPTH_TO_COLOR,       /**< Transforms depth image into the geometry of the color camera */
+    K4A_TRANSFORM_ENGINE_TYPE_DEPTH_AUX8_TO_COLOR,  /**< Transforms depth + 8 bit aux into the color camera geometry */
```

Comment:
Here are use the term AUX8 and AUX16, but the image type is MONO8 and MONO16. I think we should align these terms or at least tie them together with a comment.

Reply:
I can rename these to CUSTOM8 and CUSTOM16. On a related note there was a request to rename MONO8 and MONO16 to CUSTOM8 and CUSTOM16 in k4atypes.h.

Reply:
ok